docs/source/api/stats_summary.rst

```diff
@@ -0,0 +1,178 @@
+	|                     |          | effect. ``limit`` must be defined to make use of ``page``.                                            |
+	+---------------------+----------+-------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/stats_summary HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Last Updated Summary Stat
+""""""""""""""""""""""""""
+
+.. table:: Request Query Parameters
+
+	+-----------------+----------+---------------------------------------------------+
+	| Name            | Required | Description                                       |
+	+=================+==========+===================================================+
+	| statName        | no       | Get lastest updated date for the given stat       |
+	+-----------------+----------+---------------------------------------------------+
+	| lastSummaryDate | yes      | Tells route to get only lastest updated timestamp |
+	+-----------------+----------+---------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/stats_summary?lastSummaryDate=true HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+
+Summary Stats
+"""""""""""""
+
+:cdnName:             The CDN name for which the summary stat was taken for
+:deliveryServiceName: The :term:`Delivery Service` display name for which the summary stat was taken for
+
+	.. note:: If the ``deliveryServiceName`` is equal to ``all`` it represents summary_stats across all delivery services within the given CDN
+
+:statName:            Stat name summary stat represents
+:statValue:           Summary stat value
+:summaryTime:         Timestamp of summary, in an ISO-like format
+:statDate:            Date stat was taken, in :rfc:`3339` format
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: dHNip9kpTGGS1w39/fWcFehNktgmXZus8XaufnmDpv0PyG/3fK/KfoCO3ZOj9V74/CCffps7doEygWeL/xRtKA==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 10 Dec 2018 20:56:59 GMT
+	Content-Length: 150
+
+	{ "response": [
+		{
+            "cdnName": "CDN-in-a-Box",
+            "deliveryServiceName": "all",
+            "statName": "daily_maxgbps",
+            "statValue": 5,
+            "summaryTime": "2019-11-19 00:04:06+00",
+            "statDate": "2019-11-19
```

Comment:
You're missing a closing quote here, which is breaking syntax highlighting - because lexical analysis is failing.

But also this code block is mixing indentation with alignment (i.e. spaces and tabs). It's fine to indent the actual contained code with spaces, but for consistency the content of a `.. code-block` (or any other directive) should be tab-indented. Because of that it's also best for the code therein contained to be indented using tabs, but that's up to you.